### PR TITLE
61 send a progress update (front)

### PR DESCRIPTION
**Pull request by fadeomar**
#### Issues this PR relates to
#61 
=======

_NOTE: REMEMBER NOT TO REVIEW UNLESS ALL ACCEPTANCE CRITERIA HAS BEEN TICKED OFF_

---

#### Any info the reviewer needs to know to test 
e.g. Add env vars, run npm install

---

#### Any info the team needs to consider before merged into development
e.g. Will a database need to be rebuilt (and is this ok if existing data), add env vars

---

#### Quick Pull Request Checklist

Please make sure you ensure the following has been done before marking your PR as awaiting-review. For full checklist and process review, please refer to Process Guide (#8)

- Branch is up to date with develop
- Project runs
- You can download and install locally, and run the project locally
- All existing and new features run without bugs
- Checked that features/design works across all agreed browsers + devices
- The new work has been checked with Axe accessibility checker and doesn't throw significant errors  
- Linter is used
- Code is consistent with agreed team style
- Naming of files is consistent with project 
- Tests all pass
- No console logs

### CODE REVIEW

File: client/src/api-calls/therapists-clients.js [hunk 1]
```diff
@@ -82,6 +82,20 @@ const getMyTherapy = async ({ options } = {}) => {
...
+    // const { data } = await axios.post(
+    //   `${THERAPISTS_CLIENTS_BASE}/send-update`
+    // )
+    // return data;
+    return { data: updates };
+  } catch (error) {
+    const err = handleError(error, options);
+    return { error: err };
+  }
+};
```
Comment (RamyAlshurafa) [inline client/src/api-calls/therapists-clients.js:98]: put in the progress updates file

File: client/src/validation/schemas/therapistMessage.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import { fields, createSchema, validate as _validate } from '..';
```
Comment (RamyAlshurafa) [inline client/src/validation/schemas/therapistMessage.js:1]: change file name to "therapist-message"

File: client/src/validation/schemas/documentUpdate.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import { fields, createSchema, validate as _validate } from '..';
```
Comment (RamyAlshurafa) [inline client/src/validation/schemas/documentUpdate.js:1]: rename file name to "document-update"

**Review by RamyAlshurafa — CHANGES_REQUESTED** [review]
- [ ] send update route shows as `http://localhost:3000/programmes/:1/update`, remove the `:` please

File: client/src/api-calls/therapists-clients.js (hunk 1 above)
Comment (fadeomar) [reply client/src/api-calls/therapists-clients.js:98]: done

File: client/src/validation/schemas/therapistMessage.js (hunk 2 above)
Comment (fadeomar) [reply client/src/validation/schemas/therapistMessage.js:1]: done

**Review by RamyAlshurafa — APPROVED** [review]

**Pull request merged**